man/journald.conf.xml

```diff
@@ -107,10 +107,12 @@
       <varlistentry>
         <term><varname>Compress=</varname></term>
 
-        <listitem><para>Takes a boolean value. If enabled (the
-        default), data objects that shall be stored in the journal and
-        are larger than a certain threshold are compressed before they
-        are written to the file system.</para></listitem>
+        <listitem><para>Can takes a boolean value. If enabled (the
```

Comment:
This should probably be "Can take", but it could be fixed later.

Reply:
yeah, this doesn't look like proper English